src/zseek.h

```diff
@@ -118,6 +118,16 @@ typedef struct {
     cpu_set_t *cpuset;
 } zseek_mt_param_t;
 
+/**
+ * Compression tunables
+ */
+typedef struct {
+    /** Compression level (default = 3) */
+    int compression_level;
+    /** Compression strategy (default = fast)  */
+    int strategy;
+} zseek_zstd_param_t;
```

Comment:
How about we consolidate this with `zseek_mt_param_t` to an all-encompassing `zseek_compression_param_t`?

Reply:
Ok, while at it I've also added multi-algorithm support at least in the API level. It's easier now than after people start using it.

Reply:
Good forward thinking. Thank you!